flow/layers/child_scene_layer.cc

```diff
@@ -20,6 +20,7 @@ ChildSceneLayer::ChildSceneLayer(zx_koid_t layer_id,
 void ChildSceneLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
   TRACE_EVENT0("flutter", "ChildSceneLayer::Preroll");
   set_needs_system_composite(true);
+  context->child_scene_layer_exists_below = true;
```

Comment:
Should be unneccesary; The line above handles this

Reply:
See above comment; we need both. child_scene_layer_exists_below is there so layers that are visited afterward will know that this was below them

Reply:
Removed children_need_system_compositing_.